docs/azure/intro.md

```diff
@@ -23,7 +23,7 @@ This document provides an overview of key concepts and services .NET developers
 
 **Managed services**: Azure provides some services where you provide data or information to Azure, and Azure's implementation takes the appropriate action. One example is Azure Blob Storage, where you provide files and Azure handles reading, writing, and persisting them.
 
-**Azure SDK for .NET**: Sometimes referred to as the **Azure libraries for .NET**, this collectively refers to the [NuGet packages](https://www.nuget.org/profiles/azure-sdk) you install in your project that provide various interactions and functionality with Azure services. These packages also include management libraries used to provision and administer the resources.
+**Azure SDK for .NET**: Sometimes referred to as the **Azure libraries for .NET**, this collectively refers to the [NuGet packages](packages.md) you install in your project that provide various interactions and functionality with Azure services. These packages also include management libraries used to provision and administer the resources.
```

Comment:
I'm fine with that, @jongio. There used to be a directive from somewhere above me to use the "Azure Libraries for XXX" nomenclature in the docs to draw a clear boundary between the old monolithic SDK and the way things are done currently. I don't think it ever particularly caught on and besides, it's been a minute.

```suggestion
**Azure SDK for .NET** collectively refers to the [NuGet packages](packages.md) you install in your project that provide various interactions and functionality with Azure services. These packages also include management libraries used to provision and administer the resources.
```